en/lessons/advanced/error-handling.md

```diff
@@ -11,6 +11,17 @@ This lesson will focus on interacting with the latter.
 
 {% include toc.html %}
 
+## General conventions
+
+At the moment, Elixir community has come to some conventions regarding returning errors:
```

Comment:
`At the moment, Elixir community has come to some conventions regarding returning errors:` -> `At the moment, the Elixir community has come to some conventions regarding returning errors:` (add the word `the` before `Elixir community`)